### PR TITLE
 Ensure necessary embed strings are present for each target

### PR DESCRIPTION
**Pull request by AlCutter**

### CODE REVIEW

File: .github/workflows/pr_build.yml [hunk 1]
```diff
@@ -10,6 +10,9 @@ jobs:
     env:
       TAMAGO_VERSION: 1.21.3
       TAMAGO: /usr/local/tamago-go/bin/go
+      LOG_ORIGIN: TEST-LOG
```
Comment (mhutchinson) [inline .github/workflows/pr_build.yml:13]: Maybe an origin like: `transparency.dev/armored-witness/firmware_transparency/pr_ci/0`. All caps log origins, even for a throwaway log, gives me the shudders.

Also a comment making it clear around here that this is not the real CI log would be useful.

File: .github/workflows/pr_build.yml [hunk 2]
```diff
@@ -10,6 +10,9 @@ jobs:
...
+      LOG_PRIVATE_KEY: /tmp/log.sec
```
Comment (mhutchinson) [inline .github/workflows/pr_build.yml:14]: Is there actually a file at these paths? Seems like no, in which case a comment saying that this is just to get the Makefile to pass checks would clarify this.

**Review by mhutchinson — APPROVED** [review]

File: .github/workflows/pr_build.yml (hunk 2 above)
Comment (AlCutter) [reply .github/workflows/pr_build.yml:14]: Yes, it gets created at L29-31 along with the other throwaway keys.

File: .github/workflows/pr_build.yml (hunk 1 above)
Comment (AlCutter) [reply .github/workflows/pr_build.yml:13]: sure, although I'll see if I can come up with an origin which avoids creating confusion about CI in the first place.

File: .github/workflows/pr_build.yml (hunk 2 above)
Comment (mhutchinson) [reply .github/workflows/pr_build.yml:14]: Ta, I see that now.

**Pull request merged**